b92.html

```diff
@@ -1,43 +1,22 @@
 <!DOCTYPE html>
 
 
-
 <html>
 
 
 <head>
 
```

Comment:
Remove all unnecessary empty lines, you still have some.
Also, the indentation is still not valid, it should always be like:
```
<parent>
 <child>
 </child>
</parent>
```
In your file both head and body are in the same line with html tag, which is bad.